pkgs/applications/version-management/forgejo/default.nix

```
@@ -57,7 +58,13 @@ buildGoModule rec {
     "-X 'main.Tags=${lib.concatStringsSep " " tags}'"
   ];
 
-  postInstall = ''
+  postInstall = lib.optionalString withPreCompress ''
+     # Create static gzip and brotli files
+     find public -type f -regextype posix-extended -iregex '.*\.(css|html|js|svg|ttf|txt)' | while read file; do
+       gzip -9 -n -c $file > $file.gz
+       brotli --best -f $file -o $file.br
```

Both `gzip` and `brotli` append their corresponding extensions (`.gz` and `.br`) by default.
Any particular reason you used `-n -c >` and `-f -o` instead?

Furthermore, both `gzip` and `brotli` provide the same `-<compression level int>` and `--best` flag.
Why did you go with `-9` (same as `--best`) for `gzip` but `--best` for `brotli`?

It could be simplified to

```suggestion
       gzip --best $file
       brotli --best $file
```

---

By default gzip deletes the source file.
I was recommended to add the `-n` option in one of the mastodon/peertube PRs. Now I can't find this place.

I'll change it to this one:
```
  gzip --best --keep $file
  brotli --best --keep $file
```